### PR TITLE
pricing list reuseable workflow call

### PR DESCRIPTION
**Pull request by SAHU-01**
**Description**

This PR adds a pricing list to layer5io and the data is populated by a workflow.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
@SAHU-01 
Thank you for your contribution!
Let's discuss this during the website call on Monday at 5:30 PM IST (7:00 AM CT).

Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

**Comment by l5io** [comment]
🚀 Preview for commit 0e0d3175f0b9033f6d9dae92ee1b847bbf71ffb0 at: https://66f1456cbbdedfda387ac2bb--layer5.netlify.app

File: .github/workflows/pricing-list.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Call Pricing List
```
Comment (sudhanshutech) [inline .github/workflows/pricing-list.yml:1]: @SAHU-01 will we adding this workflow to all our repos where we needed the pricing feature?

File: .github/workflows/pricing-list.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: Trigger Pricing List Workflow
```
Comment (leecalcote) [inline .github/workflows/pricing-list.yml:1]: Let's refer to this as a features list, please (in the filename, too).

File: src/sections/Pricing/generateOptions.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      byline: "Everything in Team, plus:",
+      button: ["Coming Soon", ""]
+    }
+  };
+
+  const options = Object.entries(tiers).map(([tierName, tierInfo]) => {
+    const summary = data
+      .filter(
+        (item) =>
+          item.entire_row["Subscription Tier"] === tierName &&
```
Comment (leecalcote) [inline src/sections/Pricing/generateOptions.js:41]: Please confirm that there is a comment in the spreadsheet on this cell letting people know where to update feature list workflows if they change the column name.

**Comment by l5io** [comment]
🚀 Preview for commit 468751f8dd9a2a739fc2c6bb3ceaa77e1ebef4a0 at: https://672dd2fe5df160246cdc924c--layer5.netlify.app

**Comment by sudhanshutech** [comment]
did you incorporated coming soon? @SAHU-01

**Comment by sudhanshutech** [comment]
@jerensl workflow looks good?

**Comment by SAHU-01** [comment]
> did you incorporated coming soon? @SAHU-01

Yes @sudhanshutech the logic is there, in the json file if you add `Team-Beta` to any of the entries subscription tier you'll see the coming soon appears

**Comment by l5io** [comment]
🚀 Preview for commit 661ca03584cd059912724d52d591661539fc39b1 at: https://6736e730171fe5103fd8a52d--layer5.netlify.app

File: .github/workflows/feature-list.yml [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        if: steps.check-updates.outputs.has-updates == 'true'
+        uses: stefanzweifel/git-auto-commit-action@v5
+        with:
+          commit_message: "Updated feature data from source repository"
+          file_pattern: ${{ env.FEATURES_FILE }}
+          branch: master
+          commit_options: "--signoff"
+          commit_user_name: l5io
+          commit_user_email: ci@layer5.io
+          commit_author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
```
Comment (jerensl) [inline .github/workflows/feature-list.yml:87]: This github.actor is fragile, sometimes it willl refer to the person who merge this PR

Comment (SAHU-01) [reply .github/workflows/feature-list.yml:87]: What fix would you suggest for this one?

Comment (sudhanshutech) [reply .github/workflows/feature-list.yml:87]: @jerensl any reference or example?

Comment (jerensl) [reply .github/workflows/feature-list.yml:87]: https://github.com/layer5labs/meshery-extensions-packages/blob/master/.github/workflows/generate-feature-list.yml#L62

Comment (sudhanshutech) [reply .github/workflows/feature-list.yml:87]: thanks jeren, @SAHU-01 lets finish this.

Comment (SAHU-01) [reply .github/workflows/feature-list.yml]: Done, sudhanshu!

Comment (sudhanshutech) [reply .github/workflows/feature-list.yml]: lets give a final review on the call by everyone then will merge.

**Comment by l5io** [comment]
🚀 Preview for commit 6af6a42c0407a532e691c50dc40d1e3f6b2f0691 at: https://673763768b081eb3559ef928--layer5.netlify.app

**Comment by sudhanshutech** [comment]
@SAHU-01 good to go now?

**Comment by l5io** [comment]
🚀 Preview for commit 80e17d9bfeee3f8cb4223af5e2724a9a57181b36 at: https://6738aedd05718262113b9d7a--layer5.netlify.app

**Comment by SAHU-01** [comment]
> @SAHU-01 good to go now?

Yes @sudhanshutech, should I remove the dummy coming soon data? That was added for displaying the logic [here](https://github.com/layer5io/layer5/pull/5919/commits/661ca03584cd059912724d52d591661539fc39b1) .

**Review by jerensl — APPROVED** [review]

**Comment by sudhanshutech** [comment]
yes please remove so we can merge

**Comment by SAHU-01** [comment]
> yes please remove so we can merge

Sure, done.

**Comment by l5io** [comment]
🚀 Preview for commit 6774d51f93eedd8d9ee5d22e573acf4cb2a9685a at: https://673aa3af057182913c3b9d6b--layer5.netlify.app

**Pull request merged**